src/openrct2/ride/ride.h

```diff
@@ -219,9 +219,9 @@ typedef struct rct_ride {
 	union {
 		uint8 inversions;			// 0x114 (???X XXXX)
 		uint8 holes;				// 0x114 (???X XXXX)
-		// The undercover portion is a very rough approximation of how much of the ride is undercover.
+		// This a very rough approximation of how much of the ride is undercover.
```

Comment:
"This a" is not proper English. It should be "This is a". (Which is also what pizza2004 said.)